src/lib/global.ts

```diff
@@ -5,96 +5,160 @@ export const Global = styled.div`
     font-family: 'Helvetica Neue', sans-serif;
   }
 
-  --palette-snow-1000: #000000;
-  --palette-snow-500: #c0c0c0;
-  --palette-snow-300: #e5e5e5;
-  --palette-snow-100: #f5f5f5;
-  --palette-snow-0: #ffffff;
-
-  --palette-lavender-500: #9381f1;
-  --palette-lavender-300: #b0a3f4;
-  --palette-lavender-100: #c9c0f8;
-
-  --palette-dawn-300: #ff9097;
+  /* base colors */
+  --base-black-0: 0, 0%, 0%;
+  --base-grey-50: 247, 7%, 48%;
+  --base-white-100: 0, 0%, 100%;
+  --base-danger-color: 356, 100%, 78%;
+
+  /* grey palette */
+  --grey-lighter-1: 249, 7%, 81%;
+  --grey-lighter-2: 240, 7%, 92%;
+  --grey-lighter-3: 240, 9%, 98%;
+  --grey-lighter-4: 245, 6%, 66%;
+
+  --grey-darker-6: 248, 7%, 40%;
+  --grey-darker-7: 251, 7%, 30%;
+  --grey-darker-8: 249, 7%, 21%;
+  --grey-darker-9: 240, 7%, 11%;
```

Comment:
```suggestion
  --grey-0: 0, 0%, 0%;
  --grey-100: 249, 7%, 81%;
  --grey-200: 240, 7%, 92%;
  --grey-300: 240, 9%, 98%;
  --grey-400: 245, 6%, 66%;
  --grey-500: 247, 7%, 48%;
  --grey-600: 248, 7%, 40%;
  --grey-700: 251, 7%, 30%;
  --grey-800: 249, 7%, 21%;
  --grey-900: 240, 7%, 11%;
  --grey-1000: 0, 0%, 100%;
```

Это палитра black-white, в темной теме, все значения инвертируются и получится, что в переменной `base-black` будет лежать белый цвет, что крайне не логично и будет ломать всю концепцию абстрактных имен.

в случае `grey-0` это не яркость а насыщенность цвета. Но в какую сторону насыщенность решает тема, а не название цвета